utilities/cache_dump_load_impl.cc

```diff
@@ -312,9 +292,9 @@ IOStatus CacheDumpedLoaderImpl::RestoreCacheEntriesToSecondaryCache() {
     if (!io_s.ok()) {
       break;
     }
-    // create the raw_block_content based on the information in the dump_unit
-    BlockContents raw_block_contents(
-        Slice((char*)dump_unit.value, dump_unit.value_len));
+    // create the serialized_block based on the information in the dump_unit
```

Comment:
Is serialized_block also uncompressed block?

Reply:
I think you're right. These do not include block trailer so should be called "uncompressed block" not "serialized block". I'll fix that. 👍